src/probability_distribution.jl

```diff
@@ -4,6 +4,7 @@ ProbabilityDistribution,
 Univariate,
 Multivariate,
 MatrixVariate,
+Munivariate,
```

Comment:
This type was added in order to tell FL that a node's function maps a `Multivariate` distribution to a `Univariate` distribution. FL already supports update rules with `Multivariate` input(s) and `Univariate` output.